### PR TITLE
Feature/views for admins questions

### PR DESCRIPTION
**Pull request by bogdanrymar**
Add admins views for Questions and Answers. Add internationalization

### CODE REVIEW

File: app/controllers/admins/questions_controller.rb [hunk 1]
```diff
@@ -5,12 +5,17 @@ class QuestionsController < AdminsController
     before_action :question, only: %i[edit update destroy show]
 
     def index
-      @questions = Question.all
+      # @pagy, @questions = pagy(Question.all)
```
Comment (SvitDZHUS) [inline app/controllers/admins/questions_controller.rb:8]: Hi, why do you leave these commented lines?

File: app/views/admins/questions/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <%= f.input :body %>
+    </div>
+  </div>
+
+  <div class="waves-effect waves-light btn">
+    <%= f.submit yield(:button_text), class: "btn-area" %>
+  </div>
+
+  <div class='group-buttons', style="width:100%;">
+    <%= link_to"All questions", admins_questions_path, class: 'article-button outline' %>
```
Comment (SvitDZHUS) [inline app/views/admins/questions/_form.html.erb:25]: I see the wording is hardcoded. Maybe, better to use I18n?

File: app/views/admins/questions/edit.html.erb [hunk 3]
```diff
@@ -1 +1,10 @@
+<% provide(:button_text, 'Edit') %>
 
+<div class="form-container">
+  <div class="form row">
+    <div class="col s12">
+      <h1 class="center-align">Editing Question</h1>
```
Comment (SvitDZHUS) [inline app/views/admins/questions/edit.html.erb:6]: The same is here. As I see, some wording you've hardcoded and some not.

File: app/views/admins/questions/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            <td class="field-table-user button-functions"><%= link_to fa_icon('edit', text: 'Edit'), edit_admins_question_path(question), class: "button-functions" %></td>
+            <td class="field-table-user button-functions"><%= link_to fa_icon('trash', text: 'Delete'),admins_question_path(question), method: :delete, data: { confirm: 'Are you sure?' }, class: "button-functions" %></td>
+          </tr>
+        <% end %>
+      </tbody>
+  </table>
+
+ <%== render partial: 'shared/pagination', locals: {pagy: @pagy} %>
+
+</div>
```
Comment (SvitDZHUS) [inline app/views/admins/questions/index.html.erb:40]: Minor suggestion. Try to have a consistent indentation, please.

**Review by SvitDZHUS — COMMENTED** [review]
Minor suggestions

File: app/views/admins/questions/_form.html.erb (hunk 2 above)
Comment (bogdanrymar) [reply app/views/admins/questions/_form.html.erb:25]: in progress

File: app/controllers/admins/questions_controller.rb (hunk 1 above)
Comment (mmrshk) [inline app/controllers/admins/questions_controller.rb:8]: remove commented lines

File: app/views/admins/questions/_form.html.erb (hunk 2 above)
Comment (mmrshk) [reply app/views/admins/questions/_form.html.erb:25]: `link_to"All questions"` -> `link_to "All questions"`

File: spec/requests/admins/answers_spec.rb [hunk 5]
```diff
@@ -1,12 +1,11 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-
 RSpec.describe 'Answers', type: :request do
   let!(:question) { create(:question, :question_with_answer) }
   let!(:answer) { question.answers.first }
   let(:admin) { create(:user, :super_admin) }
-
+  include Pagy::Backend
```
Comment (mmrshk) [inline spec/requests/admins/answers_spec.rb:8]: ??

Comment (mmrshk) [reply spec/requests/admins/answers_spec.rb:8]: it's a request test, why you need it?

**Review by mmrshk — CHANGES_REQUESTED** [review]
Fix comments and also please send me a photos of your FrontEnd work

**Review by mmrshk — APPROVED** [review]
lgtm

**Pull request merged**